bio/dada2/quality-profile/environment.yaml

```
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bioconductor-dada2 == 1.16.0
```

As mentioned in the other review. Unless it is necessary to pin the exact patch version, I'd unpin it:
```suggestion
  - bioconductor-dada2 == 1.16
```

---

Done in eef6c31